examples/rmq_rpc_client.py

```diff
@@ -4,9 +4,14 @@
 
 # pylint: disable=invalid-name
 
-communicator = rmq.RmqThreadCommunicator.connect(connection_params={'url': 'amqp://localhost'})
+communicator = rmq.RmqThreadCommunicator.connect(connection_params={'url': 'amqp://127.0.0.1'})
 
-# Send an RPC message
+# Send an RPC message identifier 'fib'
```

Comment:
```suggestion
# Send an RPC message with identifier 'fib'
```

Reply:
hm, on second thought I think it might already be self-explanatory enough without this comment.
What do you say?

Reply:
It is self-explanatory enough, but providing more details is never worse :p